client/secureclient/src/main/java/com/linkedin/openhouse/client/ssl/WebClientFactory.java

```diff
@@ -109,9 +109,6 @@ protected SslContext createSslContext(String truststoreLocation) {
    */
   private WebClient createWebClient(String baseUrl, String token) {
```

Comment:
```suggestion
  private WebClient createWebClient(String baseUrl) {
```
We don't need token anymore.

Reply:
Ditto, for createSecureWebClient()